server/internal/infrastructure/wallet/btc-embedded/wallet.go

```diff
@@ -298,6 +298,27 @@ func WithBitcoindZMQ(block, tx string, host, user, pass string) WalletOption {
 			time.Sleep(1 * time.Second)
 		}
 
+		estimator, err := chainfee.NewBitcoindEstimator(
+			rpcclient.ConnConfig{
+				Host: bitcoindConfig.Host,
+				User: bitcoindConfig.User,
+				Pass: bitcoindConfig.Pass,
+			},
+			"CONSERVATIVE",
```

Comment:
Why not `ECONOMICAL`?

Reply:
https://github.com/bitcoin/bitcoin/issues/30009

Reply:
copied from existing code, should we always use `ECONOMICAL`?